frontend/tests/acceptance/pages/timeLogPage.js

```diff
@@ -9,13 +10,171 @@ module.exports = {
+    open_minute: '//button[contains(@class,"MuiPickersToolbarButton-toolbarBtn")][2]//span/h3',
+    month_container: '//div[contains(@class,"MuiPickersSlideTransition-transitionContainer")]/p',
+    next_month: '//div[contains(@class,"MuiPickersCalendarHeader-switchHeader")]/button[2]//*[name()="svg"]',
+    previous_month: '//div[contains(@class,"MuiPickersCalendarHeader-switchHeader")]/button[1]//*[name()="svg"]',
+    getYear(year) {
+      return `//div[contains(@class,"MuiPickersYearSelection-container")]/div[text()="${year}"]`;
+    },
+    getMonth(month) {
+      return `//div[contains(@class,"MuiPickersClock-clock")]/span[text()="${month}"]`;
+    },
+    getDay(day) {
+      return `//div[@role="presentation"]//p[text()="${day}"]`;
+    },
+    day: '',
+    getHour(hour) {
+      return `//div[contains(@class,"MuiPickersClock-clock")]/span[text()="${hour}"]`;
+    },
+    getMinute(minute) {
+      return `//div[contains(@class,"MuiPickersClock-clock")]/span[text()="${minute}"]`;
+    },
+    getPeriod(period) {
+      return `//button//h6[contains(text(),"${period}")]`;
+    },
+  },
+  addTimeLog(log) {
+    I.waitForElement(this.elements.openaddtimelog_dialog, 5);
+    I.click(this.elements.openaddtimelog_dialog);
+    this.selectType(log.type);
+    this.selectEventTime(log);
+    this.fillNote(log.note);
+    this.clickAdd();
+  },
+  updateTimeLog(type, log) {
+    const el_update = `${this.elements.getTimeLogContext(type)}${this.elements.openedittimelog_button}`;
+    I.waitForElement(el_update, 5);
+    I.click(el_update);
+    this.selectType(log.type);
+    this.selectEventTime(log);
+    this.fillNote(log.note);
+    this.clickUpdate();
+  },
+  deleteTimeLog(type) {
+    const el_delete = `${this.elements.getTimeLogContext(type)}${this.elements.deletetimelog_button}`;
+    I.waitForElement(el_delete, 5);
+    I.click(el_delete);
+  },
+  addToCalendar(type) {
+    const el_addtocalendar = `${this.elements.getTimeLogContext(type)}${this.elements.addtocalendar_button}`;
+    I.waitForElement(el_addtocalendar, 5);
+    I.click(el_addtocalendar);
+  },
+  selectType(type) {
+    if (type) {
+      I.waitForElement(this.fields.type, 5);
+      I.click(this.fields.type);
+      I.click(this.elements.select_options + '[contains(.,"' + type + '")]');
+      I.dontSeeElement(this.elements.select_options);
+    }
+  },
+  selectEventTime(calendar) {
+    I.click(this.fields.time);
+    I.waitForElement(this.datePicker.datePicker_container, 5);
+    this.selectYear(calendar.year);
+    this.selectMonth(calendar.month);
+    this.selectDay(calendar.day);
+    this.selectHour(calendar.hour);
+    this.selectMinute(calendar.minute);
+    this.selectPeriod(calendar.period);
+    I.click(this.elements.timelog_dialog);
+  },
+  selectYear(year) {
+    I.waitForElement(this.datePicker.open_year, 5);
+    I.click(this.datePicker.open_year);
+    const el_year = this.datePicker.getYear(year);
+    I.waitForElement(el_year, 5);
```

Comment:
try
```suggestion
    I.waitForElement(this.datePicker.getYear(year), 5);
```

Reply:
just wanted to avoid writing long lines.